Dolapo47/Epic-Mail
# create test file for group route

#### What does this PR do?
- A summary of what this task is about
Create a route to fetch a specific group in the data structure 

#### Description of Task to be completed?
- A list of the major tasks you did to complete this feature/bug/chore
Created the route
created the control file

#### How should this be manually tested?
- Outlined steps on how someone else can successfully test this feature/chore/bug to ensure it works as expected
it can be tested on postman by picking the specific id you want to fetch and adding to the group path on postman
#### Any background context you want to provide?
- Any additional `heads-up` that the tester should know about (e.g `You must be an admin to test this feature`). This is optional
